pkg/proxy/userspace/roundrobin.go

```diff
@@ -245,16 +245,19 @@ func (lb *LoadBalancerRR) updateAffinityMap(svcPort proxy.ServicePortName, newEn
 
 // buildPortsToEndpointsMap builds a map of portname -> all ip:ports for that
 // portname. Expode Endpoints.Subsets[*] into this structure.
-func buildPortsToEndpointsMap(endpoints *v1.Endpoints) map[string][]hostPortPair {
-	portsToEndpoints := map[string][]hostPortPair{}
+func buildPortsToEndpointsMap(endpoints *v1.Endpoints) map[string][]string {
```

Comment:
This really needs a test to prove that it didn't change.  Also, I'm not sure why we would want to weaken the type?